Resources/Prototypes/Tiles/floors.yml

```diff
@@ -1,8 +1,10 @@
-﻿- type: tile
+
+- type: tile
   id: floor_steel
   name: steel floor
   texture: "steel"
   base_turfs:
+  - space
   - plating
```

Comment:
This is a *terrible* solution. It has the side effect of being able to build steel floors on space tiles directly.
This bug needs to be fixed by addressing the explosion code, not by changing this.

This also goes for every other `base_turfs` you changed

Reply:
Moony told me to do exactly this

Reply:
Yea so this tells me explosion code and construction code disagree on what base_turfs is supposed to do.

Reply:
Explosion code is from like 4 years ago so obviously out of date.